CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.12.2
+
+### Runtime
+
+* Check for non-nil container after match [#26049](https://github.com/docker/docker/pull/26049)
+* Add retry logic during aufs unmount [#26536](https://github.com/docker/docker/pull/26536)
+* Prevent libdevmapper from deleting /dev/mapper/docker-xxx symbolic links in RemoveDeviceDeferred [#24740](https://github.com/docker/docker/pull/24740)
+* Fixing Regression in exit codes [#26777](https://github.com/docker/docker/pull/26777)
+* Prevent stdout / stderr race condition in limitedBuffer [#26507](https://github.com/docker/docker/pull/26507)
+* Fix exec's children handling [#26874](https://github.com/docker/docker/pull/26874)
+* Fix exec form of HEALTHCHECK CMD [#26208](https://github.com/docker/docker/pull/26208)
+
+### Networking
+* Vendor libnetwork [#26879](https://github.com/docker/docker/pull/26879) [#26953](https://github.com/docker/docker/pull/26953)
+ * Avoid returning early on agent join failures [docker/libnetwork#1473](https://github.com/docker/libnetwork/pull/1473)
+
+### Swarm Mode
+
+* Do not autodetect advertise address on join [#26211](https://github.com/docker/docker/pull/26211)
+* Vendor SwarmKit [#26765](https://github.com/docker/docker/pull/26765)
+ * Bounce session after failed status update [docker/swarmkit#1539](https://github.com/docker/swarmkit/pull/1539)
+ * Fix possible deadlocks [docker/swarmkit#1537](https://github.com/docker/swarmkit/pull/1537)
+ * Update boltdb dependency [docker/swarmkit#1510](https://github.com/docker/swarmkit/pull/1510)
+ * Fix panic in portallocator and deallocate endpoints when update with none EndpointSpec [docker/swarmkit#1481](https://github.com/docker/swarmkit/pull/1481)
+ * Validate duplicate published port in ServiceSpec [docker/swarmkit#1495](https://github.com/docker/swarmkit/pull/1495)
+ * Check entire service spec when deciding whether to replace ongoing update [docker/swarmkit#1497](https://github.com/docker/swarmkit/pull/1497)
+
+### Contrib
+
+* Debian sysvinitscript one missed DOCKER->DOCKERD change [#25869](https://github.com/docker/docker/pull/25869)
+* Patch Go with fix for OSX Sierra [#26875](https://github.com/docker/docker/pull/26875)
+* Remove go SHA check in ppc64le [#24335](https://github.com/docker/docker/pull/24335)
```

Comment:
Not sure this is important enough to mention in the changelog (afaik only IBM builds for ppc64le, and they are aware of this change)